azure-pipelines.yml

```diff
@@ -17,5 +13,38 @@ pr:
 pool: staging-pool
 
 jobs:
-  - template: .pipelines/templates/unit-test.yaml
+  - job: scan_images
+    timeoutInMinutes: 10
+    workspace:
+      clean: all
+    variables:
+      - name: TRIVY_VERSION
+        value: 0.19.2
+    steps:
+      - template: .pipelines/templates/scan-images.yaml
+  - job: lint
+    timeoutInMinutes: 5
+    workspace:
+      clean: all
+    steps:
+      - script: make lint
+        displayName: golangci-lint
+      - script: make helm-lint
+        displayName: helm lint
+  - job: build
+    timeoutInMinutes: 5
+    workspace:
+      clean: all
+    steps:
+      - script: make build build-windows
+        displayName: build
+  - job: unit_test
+    timeoutInMinutes: 5
+    workspace:
+      clean: all
+    steps:
+      - script: make unit-test
+        displayName: unit test
+      - script: bash <(curl -s https://codecov.io/bash) -C $(Build.SourceVersion)
+        displayName: Upload coverage to codecov
```

Comment:
Can we not use our existing templates in these jobs and then have it `dependsOn`? Hoping to avoid dups.